Automattic/pocket-casts-android
# Add "What's New" widgets animation

## Description

This PR adds animation to widgets "What's New" popup.

Internal ref: p1715365422718049/1715000821.182819-slack-C05RR9P9RAT

## Testing Instructions

1. Apply [this patch](https://github.com/Automattic/pocket-casts-android/files/15293960/whats_new.patch).
2. Open the app.
3. Verify that the "What's New" popup is animated.

## Screenshots or Screencast 

### Preview

| Dark mode | Light mode | Small device |
| - | - | - |
| ![Screenshot_20240513-142057](https://github.com/Automattic/pocket-casts-android/assets/30936061/617c8796-94f0-47ea-888a-be48ede06780) | ![Screenshot_20240513-142114](https://github.com/Automattic/pocket-casts-android/assets/30936061/e6233c43-22d2-40dd-82c9-44b0ff867cff) | ![Screenshot_20240513_163740](https://github.com/Automattic/pocket-casts-android/assets/30936061/ea9509ce-3e22-447e-a851-b722926c4d88) |

### Animation

![screen-20240513-162638-ezgif com-video-to-gif-converter](https://github.com/Automattic/pocket-casts-android/assets/30936061/8dceb407-5680-41b3-8377-afe928fd5c0d)

## Checklist
 
#### I have tested any UI changes...
